### PR TITLE
Add AccountClient and code for getBalance.

### PR DESCRIPTION
**Pull request by judy2k**
## Contribution Checklist
* [x] Unit tests!
* [x] Updated [CHANGELOG.md](CHANGELOG.md)
* [x] My name is in [CONTRIBUTORS.md](CONTRIBUTORS.md)

### CODE REVIEW

**Comment by judy2k** [comment]
This resolves #106

File: src/main/java/com/nexmo/client/account/BalanceRequest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.nexmo.client.account;
+
+public class BalanceRequest {
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/account/BalanceRequest.java]: Why is this an empty class? If it's not needed can we replace this with null or void in BalanceEndpoint?

File: src/test/java/com/nexmo/client/account/AccountClientTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @Before
+    public void setUp() {
+        client = new AccountClient(null);
+        client.balance = mock(BalanceEndpoint.class);
+    }
+
+    @Test
+    public void testGetBalance() throws Exception {
+        client.getBalance();
+        verify(client.balance).execute();
```
Comment (ChrisGuzman) [inline src/test/java/com/nexmo/client/account/AccountClientTest.java]: Should we add some more tests in the future or do you think this covers what's needed?

Comment (judy2k) [reply src/test/java/com/nexmo/client/account/AccountClientTest.java]: This is just the test for AccountClient - and AccountClient's job is just to pass on the request to BalanceEndpoint - which has its own tests for functionality, below.

Comment (ChrisGuzman) [reply src/test/java/com/nexmo/client/account/AccountClientTest.java]: Alrighty

File: src/main/java/com/nexmo/client/account/BalanceRequest.java (hunk 1 above)
Comment (ChrisGuzman) [reply src/main/java/com/nexmo/client/account/BalanceRequest.java]: @judy2k any thoughts on this?

Comment (ChrisGuzman) [reply src/main/java/com/nexmo/client/account/BalanceRequest.java]: @judy2k I made a PR against this PR with with my suggestion on replacing the need for this class with void  https://github.com/Nexmo/nexmo-java/pull/117

**Pull request merged**